app/Services/Publisher/SiteCodeGenerator.php

```diff
@@ -253,4 +269,19 @@ private static function getFallback(?SiteCodeConfig $config): string
 
         return "\n\t<style type=\"app/backfill\">\n{$fallback}\t</style>\n";
     }
+
+    public static function generateCryptovoxels(UrlInterface $adserverUrl, WalletAddress $walletAddress): string
+    {
+        $search = [
+            '{PAYOUT_NETWORK}',
+            '{PAYOUT_ADDRESS}',
+            '{SERVER_URL}',
+        ];
+        $replace = [
+            $walletAddress->getNetwork(),
+            $walletAddress->getAddress(),
+            (string)$adserverUrl
+        ];
+        return str_replace($search, $replace, self::CODE_TEMPLATE_CRYPTOVOXELS);
```

Comment:
Szablon powinien zostać w osobnym pliku, który tutaj powinien zostać wczytany - file_get_contents.
Sam plik szablonu będzie linkowany w instrukcji na blogu.

Reply:
Przywrócę plik resources/js/cryptovoxels/template.js i wczytam. Sprawdzę, czy `base_path()` zadziała.

Reply:
Działa to, co napisałem powyżej. Powyższy plik można wykorzystać w instrukcji.